src/3p-frame.js

```diff
@@ -75,7 +72,7 @@ function getFrameAttributes(parentWindow, element, opt_type) {
     referrer: viewer.getUnconfirmedReferrerUrl(),
     canonicalUrl: docInfo.canonicalUrl,
     pageViewId: docInfo.pageViewId,
-    clientId: element.getAttribute('ampcid'),
+    clientId: null,
```

Comment:
I think it should be safe to delete. The users are checking with
`if (global.context.clientId)`